data/en/arrayclear.json

```diff
@@ -23,19 +23,22 @@
 			"title": "Clear the value of an array",
 			"description": "Uses the arrayClear function to clear the value of an array",
 			"code": "someArray = [\"Red\", \"White\", \"Green\", \"Blue\", \"Pink\"];\nwriteOutput(arrayClear(someArray));",
-			"result": "Yes"
+			"result": "Yes",
```

Comment:
I'm not sure, if it really returns "Yes" here whereas it returns "true" for the next example … oO

Reply:
Either way, it resolves as true. Though, I agree with you that "true" is preferable.